lib/aggregators/index.js

```diff
@@ -30,7 +30,7 @@ class Aggregator {
 
 		style = style || this._printer;
 
-		if (style === 'qs' || style === 'qo' || style === 'url') {
+		if (style === 'qs' || style === 'qo' || style === 'url' || style === 'explorer-url') {
```

Comment:
I'm not sure this needs to be a printer. The output of the other non-data-fetching printers is available directly via methods in javascript. They're included in printer (with the exception of `url`, which can be used to e.g. pipe into `curl`) mainly because they were useful when debugging.